app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -0,0 +1,318 @@
+
+Example 1: You attend a job interview in England under a Standard Visitor visa in May 2023. You return home, and learn you have been given the job, which starts in September 2023. You apply for a Skilled worker visa in June 2023, 3 months before the start of your job. You move to England in July 2023. You will still be eligible for the IRP. 
+
+Example 2: You have a UK Ancestry visa and use it to visit relations in England in February 2023. You return home and successfully apply for a place on a teacher training course starting in September 2023. You move to England in June 2023. You will still be eligible for the IRP. 
+
+You will also still be eligible for the IRP if you live in another part of the UK (for example, Scotland or Wales) as long as you are employed in a school in England.  
+
+If you are in any doubt about meeting the 3-month requirement, please email us for advice before planning your move. 
+
+### Subject eligibility  
+
+The international relocation payment (IRP) has been limited to languages and physics because non-UK teachers play an important part in teaching these priority subjects. The Department for Education is piloting the IRP to encourage more non-UK teachers and trainees to bring their skills, experience and potential to English schools. 
+
+### Payment 
+
+Eligibility requirement checks can only be completed after you’ve arrived in England. Therefore, the international relocation payment (IRP) cannot be made before you take up your teacher training place or teaching job. 
+
+In order to be paid the IRP, you must have a UK bank account (this can be digital). If you are a teacher or a salaried teacher trainee, you must also have a [UK National Insurance number](https://www.gov.uk/apply-national-insurance-number). You can apply for this once you arrive in England, as long as your visa gives you the right to work and you have a confirmed job.  
+
+You will receive a single payment from the Department for Education around the end of your first term of training or employment. You will be paid as and when your eligibility requirements have been checked – we cannot guarantee an exact payment date.  
+
+You will not have to justify any expenses or give us receipts or any other evidence of the relocation costs you have incurred.  
+
+The Department for Education reserves the right to withhold payment if you are no longer studying on an eligible course or employed in an eligible job on the payment date. 
+
+### Tax 
+
+The Department for Education will pay the UK tax owed on the international relocation payment to ensure you receive the full £10,000, as long as your overall earnings do not exceed £50,270 a year in 2023/24. You may have to pay higher rate [UK Income Tax](https://www.gov.uk/browse/tax/income-tax) and [UK National Insurance](https://www.gov.uk/national-insurance) on any earnings which take you over £50,270. 
```

Comment:
```suggestion
You will receive the full £10,000, as long as your overall earnings do not exceed £50,270 a year in 2023/24. You may have to pay higher rate [UK Income Tax](https://www.gov.uk/browse/tax/income-tax) and [UK National Insurance](https://www.gov.uk/national-insurance) on any earnings which take you over £50,270. 
```
Could this be simplified, maybe like this? (If this is right?)

Reply:
I'm not sure a shorter version is clearer in this case because I think you need to understand you're being presented info about tax. The shorter version sounds like DfE will mysteriously penalise you for earning more than £50,270.